x/rollapp/keeper/rollapp_suite_test.go

```diff
@@ -57,7 +58,11 @@ func (suite *RollappTestSuite) SetupTest(deployerWhitelist ...types.DeployerPara
 	suite.Require().NoError(err)
 	err = app.BankKeeper.SetParams(ctx, banktypes.DefaultParams())
 	suite.Require().NoError(err)
-	app.RollappKeeper.SetParams(ctx, types.NewParams(true, 2, deployerWhitelist))
+	app.RollappKeeper.SetParams(ctx, types.DefaultParams())
```

Comment:
we don't have this whitelist stuff anymore, do we?

Reply:
merge conflict related
fixed